typings/semver/__init__.pyi

```diff
@@ -0,0 +1,401 @@
+"""
+This type stub file was generated by pyright.
+"""
+
+import argparse
+from __future__ import print_function
+from typing import (
+    Any,
+    Dict,
+    Generator,
+    List,
+    Literal,
+    Optional,
+    OrderedDict,
+    Tuple,
+    Union,
+)
+
+"""Python helper for Semantic Versioning (http://semver.org/)"""
+PY2 = ...
+PY3 = ...
+__version__ = ...
+__author__ = ...
+__author_email__ = ...
+__maintainer__ = ...
+__maintainer_email__ = ...
+SEMVER_SPEC_VERSION = ...
+
+def comparator(operator: Any) -> Any:  # -> (self: Unknown, other: Unknown) -> Unknown:
+    """Wrap a VersionInfo binary op method in a type-check."""
+    ...
+
+class VersionInfo:
```

Comment:
What about this -> https://github.com/python-semver/python-semver/issues/213 ?

Reply:
Do you mean to say there will be a slight impact on performance?

Reply:
Even with a poetry update?

Reply:
• Updating typing-inspect (0.6.0 -> 0.7.1)

Reply:
Somehow this still didn't work and showed lot of errors.  Merging this PR but we can revisit this if find a way to make it work with semver.py in the above link.